DIPs/DIP-42.md

```diff
@@ -0,0 +1,145 @@
+---
+DIP: 42
+Title: Backpocket, MPC for human connection using NFC social graphs
+Status: Draft
+Themes: Social, Purchases & ID, Art & Beauty
+Tags: Event Production, Software
+Authors: vivek@cursive.team, andrew@cursive.team, rachel@cursive.team
+Instances: Devcon 7 in SEA
+Resources Required: Communication support, Operational support
+Discussion: https://forum.devcon.org/t/dip-42-backpocket-mpc-for-human-connection-with-nfc-social-graphs/3667
+Created: 2024-07-09
+---
+
+## Summary of Proposal
+
+Cursive, a PSE funded R&D lab, would like to give all Devcon attendees NFC wristbands to connect with each other and check into events by tapping. The social graph created will be ported into Backpocket, an app using MPC (multi-party computation) for human connection. This includes discovering common interests, querying your connections, and visualizing communities formed through the Devcon PWA.
+
+<div align="center">
+<img src="https://forum.devcon.org/uploads/default/optimized/2X/1/11ded8a2e04f941bd3370705cb19cbcd1079a756_2_600x600.png" alt="Social Graph Illustration" width="300" height="300">
+</div>
+
+## Abstract
+
+This proposal has two components: giving every Devcon attendee an **NFC wristband**, alongside an open-source **MPC app for human connection** called Backpocket.
+
+NFC wristbands would serve three main purposes for an attendee: (1) easily share socials with other attendees, (2) tap into talks to save your Devcon experience, and (3) build a user-owned social graph of people you met in-person. Each tap produces a digital signature in Zupass [POD](https://zupass.org/pod-developers) format, backed up to Zupass to enable developers to build ZK apps using [GPCs](https://github.com/proofcarryingdata/zupass/blob/main/examples/pod-gpc-example/src/gpcExample.ts#L88).
+
+MPC apps that enable more intimate yet privacy preserving forms of human connection are a perfect fit for the NFC social graph. The three main apps would be (1) seeing common interests to find your tribe at Devcon, (2) visualizing how different tribes interact in the [Devcon PWA](https://app.devcon.org), and (3) sending out queries to friends or friends-of-friends like “Who is attending ETHBerlin ‘25 that wants to split a house?” and “Who is a ZK builder in SEA looking for a job?” for efficient and privacy-preserving discoverability.
```

Comment:
```suggestion
MPC apps that enable more intimate yet privacy preserving forms of human connection are a perfect fit for the NFC social graph. The three main apps would be (1) seeing common interests to find your tribe at Devcon, (2) visualizing how different tribes interact in the [Devcon PWA](https://app.devcon.org), and (3) sending out queries to friends or friends-of-friends like “Who is attending ETHBerlin 5 that wants to split a house?” and “Who is a ZK builder in SEA looking for a job?” for efficient and privacy-preserving discoverability.
```

probably there is no ETHBerlin in 2025 - but likely there is a ETHBerlin 5

Reply:
that's sad, i was hoping to attend next year 🥲 but sounds good, will fix!